plugin-packs/postcss-preset-env/README.md

```diff
@@ -9,9 +9,17 @@
 can understand, determining the polyfills you need based on your targeted
 browsers or runtime environments.
 
-```bash
-npm install postcss-preset-env
-```
+## How does it work?
+
+[PostCSS Preset Env] is a Plugin Pack for [PostCSS]. It leverages the list of the features we keep an eye from [CSSDB][cssdb] and applies plugins, so you can use those new features without having to worry about browser support.
```

Comment:
Should we add a small intro with keywords and what they mean in the context of this project?

Some things that could be confusing or vague :

- browser list option vs. browser support stats
- css feature vs. csstools plugin
- plugin

This will help people who are not yet familiar with polyfills and fallbacks in general while also serving as a guide for us to be consistent.

Reply:
I love that idea!